### PR TITLE
Use correct play & pause music icons

### PR DESCRIPTION
**Pull request by wvpm**

### CODE REVIEW

File: game/src/Game/GameStart.tscn [hunk 1]
```diff
@@ -57,7 +57,7 @@ process_mode = 2
 disable_3d = true
 title = "VIC2_DIR_DIALOG_TITLE"
 size = Vector2i(935, 159)
-ok_button_text = "VIC2_DIR_DIALOG_SELECT"
+ok_button_text = "Select Current Folder"
```
Comment (wvpm) [inline game/src/Game/GameStart.tscn:60]: I did not touch this. Why has this changed?

Comment (Hop311) [reply game/src/Game/GameStart.tscn:60]: This gets changed upon localisation by the Godot editor due to a bug, there's a comment explaining it in a GDScript file somewhere. Unfortunately we just have to ignore this and not include it in PRs, hopefully it'll be fixed with Godot 4.3.

**Review by Hop311 — COMMENTED** [review]
LGTM, just gotta remove that annoying localisation thing and it'll be ready to merge

File: game/src/Game/GameStart.tscn (hunk 1 above)
Comment (wvpm) [inline game/src/Game/GameStart.tscn:60]:
```suggestion
ok_button_text = "VIC2_DIR_DIALOG_SELECT"
```

**Review by Hop311 — APPROVED** [review]

**Pull request merged**